build.rs

```diff
@@ -53,9 +64,12 @@ fn main() {
         // TODO: Test if this is required on Windows too?
         println!("cargo:rustc-link-lib=ncurses");
     }
-
-    println!("cargo:rustc-link-lib=z");
-    println!("cargo:rustc-link-lib=stdc++");
+    if cfg!(linux) {
+        println!("cargo:rustc-link-lib=z");
+        println!("cargo:rustc-link-lib=stdc++");
+    } else if cfg!(windows) {
+        println!("cargo:rustc-link-lib=zlib");
```

Comment:
I was wondering why this wouldn't compile on Linux with -lzlib but apparently it's -lz on Linux. The more you know...